src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        I am
+        {` ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm'
+          ? 'wife'
```

Comment:
It's not a good idea to nest several ternary operators.
Create a variable for the partner before